packages/blockchain-wallet-v4/src/redux/payment/nfts/utils.ts

```diff
@@ -1065,7 +1065,7 @@ export async function _makeSellOrder({
   quantity: number
   startAmount: number
   waitForHighestBid: boolean
-}): Promise<UnhashedOrder> {
+}): Promise<any> {
```

Comment:
why the change to any?

Reply:
This is just for testing, because we're going from BigNumber to string, in unhashed order the baseprice type is a BigNumber